templates/CRM/Admin/Page/APIExplorer.tpl

```diff
@@ -162,7 +162,7 @@
       <label for="api-entity">{ts}Entity{/ts}:</label>
       <select class="crm-form-select big required" id="api-entity" name="entity">
         <option value="" selected="selected">{ts}Choose{/ts}...</option>
-        {crmAPI entity="Entity" action="get" var="entities" version=3}
+        {crmAPI entity="Entity" var="entities"}
```

Comment:
If you're asking if this code changes the behavior of the smarty api or the api explorer, no it doesn't, it's just code cleanup. The biggest change is to unset the correct variables in smarty_function_crmAPI (the variables are named `var` and `action` but previously it was unsetting nonexistant variables `assign` and `method`). Nothing else is changed. The api explorer will continue to put `action` in its generated code (even though that's redundant if the action is "get" but who cares).